spec/obs_github_deployments/deployment_spec.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+require "test_helper"
+
+RSpec.describe ObsGithubDeployments::Deployment, :vcr do
+  subject { ObsGithubDeployments::Deployment.new(repository: gh_test_repository, access_token: gh_test_access_token) }
+
+  describe "locked?" do
+    it "returns true when deployment state is 'queued'" do
```

Comment:
```suggestion
    it "returns true when deployment is locked" do
```

Reply:
Maybe is better if we avoid mentioning implementation details.

Reply:
done